### ISSUE TITLE
o!mania ranking criteria pattern descriptions missing context

### ISSUE THREAD

**Issue by WinterIsPretty** [ppy/osu-wiki#6231]
The osu mania ranking criteria has an issue where patterns aren't described in the right order. I believe they should be listed in the right order where a new player will be able to read from top to bottom and use the definitions they just read to understand the next patterns.

An example of the issue: Brackets are explained at the beginning and contains the word "trill" in the definition, but trill is described at the very end of the list.

How I think it should be ordered:

- Chord
- Grace
- Jump
- Hand
- Jack
- Minijack
- Anchor
- Trill
- Jumptrill
- Split-jumptrill
- Bracket
- Stream
- Chordstream (Jumpstream, Handstream, Quadstream)
- Roll
- Release
- Shield

**Comment by TicClick** [comment]
stumbled upon this when I started translating the o!m ranking criteria myself. should be possible to resolve (in a better way) once there are articles/stubs about individual patterns -- see https://github.com/ppy/osu-wiki/issues/3151#issuecomment-1366764788

**Comment by TicClick** [comment]
JFYI once the criteria are reworked @Protastic101 will probably get to it faster than I

**Comment by Protastic101** [comment]
Yep, currently the o!m RC rework is on hold while the mania NAT wait for everyone to be back from their LoAs. I think we can probably expect to see a bit more progress on this once the general o!m RC draft comes out (tentative timeline is looking to be June or July). I've made a note to include and update these things when that happens.
